src/main/java/org/broadinstitute/dsde/consent/ontology/resources/DataUseResource.java

```diff
@@ -26,7 +26,13 @@ public Response getSchema() {
         String content = FileUtils.readAllTextFromResource("data-use.json");
         return Response.ok().entity(content).type(MediaType.APPLICATION_JSON).build();
     }
-
+    @GET
+    @Path("/data-use-v3")
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response getSchemaV3() {
+        String content = FileUtils.readAllTextFromResource("data-use-v3.json");
+        return Response.ok().entity(content).type(MediaType.APPLICATION_JSON).build();
+    }
```

Comment:
Don't forget that we need an entry for this in api-docs.yaml

Reply:
Do I need to update everything with 'data-use' in the path to 'data-use-v3', or just add an additional path for ' /schemas/data-use-v3'?

Reply:
Just add a new path and leave the existing one in place